src/seedu/addressbook/data/person/Address.java

```diff
@@ -1,18 +1,32 @@
 package seedu.addressbook.data.person;
 
 import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.address.Block;
+import seedu.addressbook.data.person.address.Street;
+import seedu.addressbook.data.person.address.Unit;
+import seedu.addressbook.data.person.address.PostalCode;
 
 /**
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
 public class Address {
+    public static final String EXAMPLE = "123, some street, #12-34, 123456";
+    public static final String ADDRESS_FORMAT = "BLOCK, STREET, UNIT, POSTAL CODE";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in format " + ADDRESS_FORMAT;
+    public static final String ADDRESS_VALIDATION_REGEX = ".+,.+,.+,.+";
```

Comment:
Quite okay but sometimes address may not have a valid block etc, is it wise to strictly enforce all fields must be entered?